### DISCUSSION TITLE
Any Way to Bind Alternate Keyboard Shortcuts?

### DISCUSSION THREAD

**Discussion by RenStimpy** [WinMerge/winmerge#758] [Q&A]
WinMerge seems perfect for our needs. The only gotcha is that everyone around here is deeply ingrained with using the old tool's keyboard shortcuts for some very basic operations (e.g. next/previous delta). And old dogs really dislike learning new tricks. So I'm hoping there's a way, even via a manually edited config file, that we can remap a few of the keyboard shortcuts. Is this possible, and if so, where should I be looking?

Thanks in advance for any helpful pointers!

**Comment by t-rapp** [comment]
Also stumbled over the changed keyboard shortcuts when migrating to a new computer installation and looked for a way to configure shortcuts.

In my case I find it difficult to get used to the new "one key" shortcuts like those for cursor movement, which now seem to trigger the same function like the original "two key" combination (move to next/prev difference, copy to right/left). Holding down the Alt key is not necessary anymore, but the original key function is lost that way.

**Reply by sdottaka** [reply]
Perhaps you have merge mode enabled. If merge mode is enabled, you do not need to press the Alt key; pressing the Down key will move to the next difference.
Pressing the F9 key or clicking the File → Merge Mode menu item toggles the merge mode.

**Reply by t-rapp** [reply]
Yes, toggling off Merge Mode did it. Did not know what that menu item does and overlooked it when searching through the configuration possibilities. Thanks a lot!
